Expensify/App
# Add message user button

### Details
Adds a message user button when seeing a user's profile.

### Fixed Issues
$ https://github.com/Expensify/App/issues/6661

### Tests
1. Create a group chat.
2. Click the chat header and select one of the users.
3. Verify that there is a `Message <user>` button and clicking it navigates you to the chat with that user.

### QA Steps
Steps above.

### Tested On


### Screenshots


#### Web

![Screen Shot 2022-01-11 at 4 49 26 PM](https://user-images.githubusercontent.com/22219519/149039328-b480275c-eff3-42b4-8f7e-22a5b0727228.png)


#### Mobile Web

![mweb](https://user-images.githubusercontent.com/22219519/149038586-a1ec681e-eab4-4d63-b048-d07ad08ccf87.png)

#### Desktop

<img width="1202" alt="desktop" src="https://user-images.githubusercontent.com/22219519/149038591-da46ec50-0181-436a-9c71-0a5ebfc0a23e.png">

#### iOS

![ios](https://user-images.githubusercontent.com/22219519/149038598-40952324-b0ec-46e7-b3d2-fe5ed1d4076b.png)

#### Android
![android](https://user-images.githubusercontent.com/22219519/149039019-aa7324cc-1151-4997-95ff-c072e46e68c5.png)
